locale/uk/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Рекомендовано для більшості
   changelog: Список змін
   api: Документація
-  version-schedule-prompt: Або подивіться на
-  version-schedule-prompt-link-text: графік LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-uk  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?